x-pack/legacy/plugins/uptime/public/components/functional/monitor_status_bar.tsx

```diff
@@ -93,11 +102,29 @@ export const MonitorStatusBarComponent = ({ data, monitorId }: Props) => {
                 defaultMessage: 'Time since last check',
               }
             )}
-            grow={false}
+            grow={true}
           >
             {moment(new Date(timestamp).valueOf()).fromNow()}
           </EuiFlexItem>
         </EuiFlexGroup>
+        {certificateValidity && (
+          <>
+            <EuiSpacer size="s" />
+            <EuiText
+              color="subdued"
+              grow={false}
+              size="s"
+              aria-label={i18n.translate('xpack.uptime.monitorStatusBar.sslCertificateExpiry', {
+                defaultMessage: 'SSL certificate expires',
+              })}
+            >
+              {i18n.translate('xpack.uptime.monitorStatusBar.sslCertificateExpiry', {
```

Comment:
This might be one of the first times you've encountered this particular eccentricity of the `i18n` lib, but we are supposed to specify values for the translation text in the translation. I'd recommend using the `<FormattedMessage>` component as well.

`import { FormattedMessage } from '@kbn/i18n/react';`

```suggestion
              <FormattedMessage
                id="xpack.uptime.monitorStatusBar.sslCertificateExpiry"
                defaultMessage="SSL certificate expires {certificateValdiity}"
                values={{
                  certificateValidity: moment(new Date(certificateValidity).valueOf()).fromNow(),
                }}
              />
```

Reply:
Updated